src/meta/processors/BaseProcessor.h

```diff
@@ -23,6 +23,9 @@ namespace meta {
 
 using nebula::network::NetworkUtils;
 
+const PartitionID kDefaultPartId_ = 0;
```

Comment:
Put it some where more common,  use "static const" and remove the underscore in name.

Reply:
Some processors only depends on this header file, I will rename it anyway.